src/NUnitFramework/testdata/TimeoutFixture.cs

```diff
@@ -27,7 +27,7 @@ public void TearDown()
         [Test, Timeout(50)]
         public void InfiniteLoopWith50msTimeout()
         {
-            while (true) { }
+            Thread.Sleep(5000);
```

Comment:
👍 thank you for this update too

Reply:
nitpick: Actually, I feel like this may no longer do what the test method name says. Is it worth keeping keeping one busy wait in place to simulate a CPU-bound long running test?

Reply:
I mainly made this change to prevent the builds hanging and taking up unnecessary resources.
Yes, they probably should be renamed.
I don't see a point in making a CPU-bound versions. It should not make a difference in 'interuptability'.